HMCLCore/src/main/resources/assets/gamerule/gamerule.json

```
@@ -0,0 +1,521 @@
+    "ruleKey": [
+      "minecraft:ender_pearls_vanish_on_death",
+      "enderPearlsVanishOnDeath"
+    ],
+    "displayI18nKey": "gamerule.rule.ender_pearls_vanish_on_death",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:entity_drops",
+      "doEntityDrops"
+    ],
+    "displayI18nKey": "gamerule.rule.entity_drops",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "entitiesWithPassengersCanUsePortals"
+    ],
+    "displayI18nKey": "gamerule.rule.entities_with_passengers_can_use_portals",
+    "defaultValue": false
+  },
+  {
+    "ruleKey": [
+      "minecraft:fall_damage",
+      "fallDamage"
+    ],
+    "displayI18nKey": "gamerule.rule.fall_damage",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:fire_damage",
+      "fireDamage"
+    ],
+    "displayI18nKey": "gamerule.rule.fire_damage",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:fire_spread_radius_around_player"
+    ],
+    "displayI18nKey": "gamerule.rule.fire_spread_radius_around_player",
+    "defaultValue": 128,
+    "minValue": -1
+  },
+  {
+    "ruleKey": [
+      "minecraft:forgive_dead_players",
+      "forgiveDeadPlayers"
+    ],
+    "displayI18nKey": "gamerule.rule.forgive_dead_players",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:freeze_damage",
+      "freezeDamage"
+    ],
+    "displayI18nKey": "gamerule.rule.freeze_damage",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:global_sound_events",
+      "globalSoundEvents"
+    ],
+    "displayI18nKey": "gamerule.rule.global_sound_events",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:immediate_respawn",
+      "doImmediateRespawn"
+    ],
+    "displayI18nKey": "gamerule.rule.immediate_respawn",
+    "defaultValue": false
+  },
+  {
+    "ruleKey": [
+      "minecraft:keep_inventory",
+      "keepInventory"
+    ],
+    "displayI18nKey": "gamerule.rule.keep_inventory",
+    "defaultValue": false
+  },
+  {
+    "ruleKey": [
+      "minecraft:lava_source_conversion",
+      "lavaSourceConversion"
+    ],
+    "displayI18nKey": "gamerule.rule.lava_source_conversion",
+    "defaultValue": false
+  },
+  {
+    "ruleKey": [
+      "minecraft:limited_crafting",
+      "doLimitedCrafting"
+    ],
+    "displayI18nKey": "gamerule.rule.limited_crafting",
+    "defaultValue": false
+  },
+  {
+    "ruleKey": [
+      "minecraft:locator_bar",
+      "locatorBar"
+    ],
+    "displayI18nKey": "gamerule.rule.locator_bar",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:log_admin_commands",
+      "logAdminCommands"
+    ],
+    "displayI18nKey": "gamerule.rule.log_admin_commands",
+    "defaultValue": true
+  },
+  {
+    "ruleKey": [
+      "minecraft:max_block_modifications",
+      "commandModificationBlockLimit"
+    ],
+    "displayI18nKey": "gamerule.rule.max_block_modifications",
+    "defaultValue": 32768,
+    "minValue": -1
```

在 [25w44a](https://minecraft.wiki/w/Java_Edition_25w44a) 中限制为 `1`

```suggestion
    "minValue": 1
```